fbw-a32nx/src/systems/fmgc/src/guidance/geometry/GeometryFactory.ts

```diff
@@ -30,26 +30,13 @@ import { PILeg } from '@fmgc/guidance/lnav/legs/PI';
 import { FMLeg } from '@fmgc/guidance/lnav/legs/FM';
 import { FALeg } from '@fmgc/guidance/lnav/legs/FA';
 import { HALeg, HFLeg, HMLeg } from '../lnav/legs/HX';
-
-function getFacilities(): typeof Facilities {
-  if ('Facilities' in window) {
-    return Facilities;
-  }
-
-  return {
-    getMagVar(_lat: Degrees, _long: Degrees): Degrees {
-      return 0;
-    },
-  };
-}
+import { MagVar } from '@microsoft/msfs-sdk';
```

Comment:
We should not use the msfs sdk class. We want to switch to our own mag model using the same DB as the real HWL FMS later. We can just directly use Facilities.getMagVar and there's one less thing to search/replace.